docs/roadmap.md

```diff
@@ -29,29 +36,26 @@ While we don't intend to break existing functionality, we will likely deprecate
 
 ### Future Features
 We're exploring how to support these improvements
-* Column Virtualization (v2.0)
-* Frozen rows (v2.0)
-* Flexible styling / themes (v2.0)
+* Column Virtualization (v3.0)
+* Frozen rows (v3.0 or later)
+* Flexible styling / themes (v3.0 or later)
 * Multi-row / multi-column cells (later)
 
 ### Other Improvements
 * Additional unit testing
-* ES6 & styling improvements
 * Cleanup build system
 * More examples demonstrating common use cases
 
 Timeline and dependencies
 ---------------
 
-### v2.0
-Targeted for Q3 2019  
+### v3.0
+Targeted for Q4 2023
```

Comment:
Not sure about the timeline here, so I'm just keeping to 2023 Q4 until we figure a more accurate time.